hummedia/api.py

```diff
@@ -35,6 +35,17 @@ def Collection(collection,id=None):
 	else:
 		return endpoint_404()
 
+@app.route('/video/stale', methods=['GET', 'OPTIONS'])
+@app.route('/video/stale/<date>', methods=['GET', 'OPTIONS'])
+@crossdomain(origin=CROSS_DOMAIN_HOSTS,headers=['Origin','x-requested-with','accept','Content-Type'],credentials=True)
+def getStaleMedia(date=None):
+    return MediaAsset(request).get_stale(date)
+
+@app.route('/video/<id>/view', methods=['GET', 'PUT', 'OPTIONS'])
```

Comment:
I'm actually using the PUT request from the client to update the view date. The GET request is left over from me debugging in the browser. I'll remove that now.